utils/templates.py

```diff
@@ -261,3 +261,55 @@ def __init__(self, params=None, content=None):
 
         self.body = copy.deepcopy(self.params)
         self.body.update({'content': self.content})
+
+class ShortReviewReply(InvitationReply):
+    """
```

Comment:
actually, for simplicity, let's just leave it here. it will be removed when i make my other pull request.

@melisabok @pmandler is that ok?